Add contentbytes promo (#124)
techwriter-pl/techwriter-pl.github.io: blog/content-bytes-pazdziernik-2024/index.md

---

> title: 'ContentBytes#1: Nowe wydarzenie na contentowej mapie!'

Reviewer: ```suggestion
title: 'ContentBytes: Nowe wydarzenie na contentowej mapie!'
```
W sumie numer nie jest częścią nazwy

authors: bczyz
date: '2024-09-26'
tags:
  - 'news'
  - 'meetup'
  - 'spolecznosc'
coverImage: 'ContentBytesKrk1.png'
---

> # ContentBytes#1: Nowe wydarzenie na contentowej mapie!

Reviewer: ```suggestion
# ContentBytes: Nowe wydarzenie na contentowej mapie!
```

Upalne lato już za nami, ale ekipa **ContentBytes** nie próżnuje, dbając o skuteczne podgrzanie atmosfery tej jesieni.
Już za tydzień mamy dla was nie jeden, a dwa tematy :)
Podczas spotkania porozmawiamy o dokumentacji API oraz przyszłości twórców contentu w świecie dynamicznie zmieniających się produktów.

> Ugości nas **Forte Spaces**, a dyskusje uprzyjemnią nam przekąski :pizza:.

Reviewer: ```suggestion
Ugości nas **Forte Spaces**, a dyskusje uprzyjemnią nam przekąski :pizza: od **Apify**.
```
Jak promować to promować?

Tematy przybliżą wam [**Barbara Czyż**](https://www.linkedin.com/in/barbara-szwarc) oraz [**Paweł Chłodnicki**](https://www.linkedin.com/in/pawelchlodnicki/).

<!--truncate-->

Pamiętajcie o rejestracji, to pozwoli nam na wybór odpowiedniej sali i
zaplanowanie poczęstunku. Link do formularza znajdziecie
[tutaj](https://forms.gle/NzEDRbvS1gWDZY3D8).

## Tematy

**_Dokumentacja API: Z czym to się je?_**

Dokumentacja API to temat, wokół którego krąży obecnie wiele pytań:

* Kto powinien dokumentować API?
* Jak zaplanować content, żeby dokumentacja była pomocna i sensownie ustrukturyzowana?
* Jakimi standardami powinni się kierować technical writerzy?

Opierając się na przykładach, omówimy proces, w którym pomysł na nowe API staje się udokumentowaną specyfikacją.

> Podczas prezentacji zapoznacie się z narzędziami, praktykami, oraz technikami współpracy, które pomagają nam tworzyć dokumentację do API.

Reviewer: ```suggestion
Podczas prezentacji zapoznacie się z narzędziami, praktykami oraz technikami współpracy, które pomagają nam tworzyć dokumentację do API.
```

**_Rola i przyszłość twórców contentu w tworzeniu produktu_**

Pojęcie **content** przywołuje różne skojarzenia, często zależne od kontekstu, w którym zostało użyte.
W swojej prezentacji, Paweł opowie nam jak content jest postrzegany przez pryzmat firmy, zespołu, momentu w rozwoju produktu, a także wcześniej wspomnianego **kontekstu**.
Wspólnie zastanowimy się, jak te okoliczności mogą mieć pozytywny wpływ zarówno na twórców, jak i odbiorców tworzonych treści.

## Prelegenci

> Podczas pierwszego spotkania ContentBytes, prezentacje poprowadzą:

Reviewer: ```suggestion
Podczas pierwszego spotkania ContentBytes prezentacje poprowadzą:
```

Krupnicza 3, Kraków

## Kilka słów o sponsorach

Pierwsze spotkanie ContentBytes jest sponsorowane przez **Forte Digital** i **Apify**.

### Forte Digital

W DNA Forte Digital znajdziesz pierwiastek polski, skandynawski i oczywiście
technologiczny. Forte Digital powstało w 2017 roku jako owoc udanej współpracy
polskich i norweskich założycieli. Od tego czasu z powodzeniem czerpie to, co
najlepsze ze swoich kultur, a od niedawna w swoich szeregach ma koleżanki i
kolegów z Danii oraz Niemiec. Forte Digital skupia się najczęściej na projektach
dotyczących obszaru e-commerce w pełnym zakresie realizacji – development i
consulting IT oraz design.

### Apify

[Apify](https://apify.com/) jest platformą, którą programiści mogą wykorzystać
do budowania, wdrażania i publikowania narzędzi do web scrapingu, ekstrakcji
danych i automatyzacji. Apify udostępnia ponad 1500 gotowych narzędzi,
pozwalając na uzyskanie dodatkowego przychodu poprzez udostępnianie je innym
użytkownikom. Dodatkowo Apify odpowiada za [Crawlee](https://crawlee.dev/),
publiczną bibliotekę do web scrapingu.

## O nas

Celem **ContentBytes** jest stworzenie przestrzeni, w której twórcy contentu

> mogą wymienić się doświadczeniami i wspólnie poruszyć na tematy związane z tworzeniem treści.

Reviewer: ```suggestion
mogą wymieniać się doświadczeniami i omawiać tematy związane z tworzeniem treści.
```

> Uważamy, że content to nie tylko tworzenie dokumentacji, ale cały ekosystem uwzględniający UX, UI, tworzenie materiałów szkoleniowych i prezentacji, a także tematyka związana z dostępnością i designem.

Reviewer: ```suggestion
Według nas, content to nie tylko sam proces pisania, ale tak naprawdę cały ekosystem uwzględniający UX, UI, design, a także zagadnienia dostępności, zarządzania i związanej z nimi technologii.
```
